Frameworks/SPMySQLFramework/SPMySQLFramework.xcodeproj/project.pbxproj

```diff
@@ -732,7 +731,6 @@
 				GENERATE_PKGINFO_FILE = YES;
 				INFOPLIST_FILE = Resources/Info.plist;
 				INSTALL_PATH = "@executable_path/../Frameworks";
-				LD_DYLIB_INSTALL_NAME = "$(DYLIB_INSTALL_NAME_BASE:standardizepath)/$(EXECUTABLE_PATH)";
```

Comment:
Why get rid of this line @Kaspik?

Reply:
Because it's tha same in "super-target" so there is no need of inheritance. :)